docs/workflow_tasks/discover_granules.md

```diff
@@ -50,6 +50,26 @@ A list of buckets with types that will be used to assign bucket targets based on
 
 A Cumulus [collection](https://github.com/nasa/cumulus/blob/master/packages/api/models/schemas.js) object.    Used to define granule file groupings and granule metadata for discovered files.   The collection object utilizes the collection type key to generate types in the output object on discovery.
 
+#### Ignore Files Configuration (`ignoreFilesConfigForDiscovery`)
+
+The `boolean` property `ignoreFilesConfigForDiscovery` indicates whether or not
```

Comment:
Maybe it's worth to indicate this property is optional.

Reply:
Since the next sentence says, "By default, this property is `false`, ..." I thought that would be implied. If you think that's not clear enough, I'll add an explicit optional indication.